runtime/rococo/src/lib.rs

```diff
@@ -94,14 +117,14 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("rococo"),
 	impl_name: create_runtime_str!("parity-rococo-v2.0"),
-	authoring_version: 0,
+	authoring_version: 0, // TODO: Params -> authoring_version: 2
 	spec_version: 9260,
```

Comment:
`sped_version` needs a bump. I would do `9261`

Reply:
It will be updated accordingly with last `master` version